swarmrl/networks/flax_network.py

```diff
@@ -136,40 +139,52 @@ def compute_action(self, feature_vector: np.ndarray, explore_mode: bool = False)
 
         return index, model_output[index]
 
-    def export_model(self, directory: str = "Models"):
+    def export_model(self, filename: str = "model", directory: str = "Models"):
```

Comment:
What does the file structure look like for multiple models? Have you checked the Gym to make sure this will work as well as all of the notebooks? I guess this will come up in the CI when it is running properly.